lib/rubocop/cop/rspec/each.rb

```diff
@@ -0,0 +1,53 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that all matcher is used instead of iterating over an array.
+      #
+      # @example
+      #   # bad
+      #   it 'validates users' do
+      #     [user1, user2, user3].each { |user| expect(user).to be_valid }
+      #   end
+      #
+      #   # good
+      #   it 'validates users' do
+      #     expect([user1, user2, user3]).to all(be_valid)
+      #   end
+      class Each < Cop
```

Comment:
Maybe `AllMatcher`? I usually like looking at the rspec internal naming but it isn't super helpful in this case heh https://github.com/rspec/rspec-expectations/blob/82ab90b513dfcd0b52064f56be4b2c6218adeded/lib/rspec/matchers.rb#L638-L655